tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -964,7 +933,6 @@ describe('main_html5 wrapper tests', function () {
   });
 
   it('should not raise time update events while priming', function(){
-    vtc.interface.EVENTS.TIME_UPDATE = "timeupdate";
```

Comment:
lol sorry, I didn't realize you'd have to update all of these. Should make it easier to add new tests though.